JuiceMaker/JuiceMaker/Controller/ViewController.swift

```diff
@@ -1,17 +1,149 @@
 //  JuiceMaker - ViewController.swift
-//  Created by yagom. 
+//  Created by yagom.
 //  Copyright © yagom academy. All rights reserved.
 
 import UIKit
 
 final class ViewController: UIViewController {
     private let juiceMaker = JuiceMaker()
+    private var fruitsStock: [Fruits: Int] {
+        return FruitStore.shared.fruitsStock
+    }
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        juiceMaker.make(juice: .strawberryBanana)
+        displayStock()
+    }
+    
+    @IBOutlet weak var stockOfStrawberry: UILabel!
+    @IBOutlet weak var stockOfBanana: UILabel!
+    @IBOutlet weak var stockOfPineApple: UILabel!
+    @IBOutlet weak var stockOfKiwi: UILabel!
+    @IBOutlet weak var stockOfMango: UILabel!
+    
+    func displayStock() {
+        if let strawberryStock = fruitsStock[.strawberry],
+           let bananaStock = fruitsStock[.banana],
+           let pineappleStock = fruitsStock[.pineapple],
+           let kiwiStock = fruitsStock[.kiwi],
+           let mangoStock = fruitsStock[.mango] {
+            stockOfStrawberry.text = String(strawberryStock)
+            stockOfBanana.text = String(bananaStock)
+            stockOfPineApple.text = String(pineappleStock)
+            stockOfKiwi.text = String(kiwiStock)
+            stockOfMango.text = String(mangoStock)
+        }
     }
     
+    @IBAction func ClickChangeStock(_ sender: Any) {
+        moveToChangeStockViewController()
+    }
+    
+    func moveToChangeStockViewController() {
+        guard let nextVC = self.storyboard?.instantiateViewController(withIdentifier: "ChangeStock") as? ChangeStockViewController else { return }
+        self.navigationController?.pushViewController(nextVC, animated: false)
+    }
+    
+    func setSuccessAlert(juiceName: JuiceMaker.Juice) {
+        let successAlert = UIAlertController(title: "\(juiceName.name)가 나왔습니다 맛있게 드세요", message: nil, preferredStyle: UIAlertController.Style.alert)
+        let offAction = UIAlertAction(title: "확인", style: UIAlertAction.Style.default, handler: nil)
+        successAlert.addAction(offAction)
+        present(successAlert, animated: true, completion: nil)
+    }
```

Comment:
```suggestion
    func setSuccessAlert(juiceName: JuiceMaker.Juice) {
        let successAlert = UIAlertController(
            title: "\(juiceName.name)가 나왔습니다 맛있게 드세요",
            message: nil, 
            preferredStyle: UIAlertController.Style.alert
        )
        let offAction = UIAlertAction(
            title: "확인", 
            style: UIAlertAction.Style.default,
            handler: nil
        )
        successAlert.addAction(offAction)
        present(successAlert, animated: true, completion: nil)
    }
```

보통 파라미터가 3개 이상일 때는 이런 식으로 개행을 해주기도 합니다 ☺️ 스타일이니 참고만 해주세요

Reply:
파라미터명이 juiceName이라 사용할 때 juiceName.name 으로 나오는 것이 어색한 것 같아요 파라미터 명을 수정해볼까요?

Reply:
juiceName.name 대신 juice.name으로 변경했습니다!

Reply:
name의 중복 사용을 피하기 위해 파라미터를 ```juice```로 변경했습니다!
title, message, preferredStyle도 개행을 적용해봤어요🧐
개행하는게 가독성이 더 좋아보이네요!!